### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Pierric Ripoll, Victor Nicolet and Colin Moschard

### DISCUSSION THREAD

**Discussion by Nyaaw** [heig-vd-dai-course/heig-vd-dai-course#359] [Show and tell]
Our idea is to quickly edit some tags of audio files.

We intend to use the library [JAudiotagger](https://jthink.net/jaudiotagger/).

The tool will be able to open a file, display the original tags, and change a selection of tags. 
If the tags are changed, the tool saves a copy of the file with the new tags in.

At first, the tags supported for edit will be
- Track title
- Album name
- Artist name
- Year of track

At first, we intend to **only support files in the [FLAC format](https://xiph.org/flac/)**, and if we finish early, to add support for other file formats (mp3, ogg, wav)

**Link to the repository**:
[colinmoschard/TagEdit](https://github.com/colinmoschard/TagEdit)

**Comment by ludelafo** [comment]
Looks good for us! Please be wary of the time available for the project. Go to the essentials and add more features if the time allows it.

Also, the repository seems private? We aren't able to access it. Please invite us if you want to keep your project private.

**Reply by Nyaaw** [reply]
We invited yo to the repo :)

**Comment by Nyaaw** [comment]
Rendu final :

repo: https://github.com/colinmoschard/TagEdit
branche main
commit hash n° d94c7d0dba367fb35c3b774c826bcf471408bf37

Nous venons de remarquer que nous avions oublié qu'il fallait vous envoyer un message pour vous informer du rendu, nous en sommes désolés... 

Cependant, le commit est bien daté d'hier, avant la deadline. 

J'espère que vous serez compréhensifs car on ne veut vraiment pas prendre un point sur la note pour une petite erreur aussi bête 😓
